app/javascript/components/NavBar.vue

```diff
@@ -8,7 +8,7 @@
 
     <template slot="start">
       <b-navbar-item href="/about">About</b-navbar-item>
-      <b-navbar-item href="/contributions">Contributions [ADMIN/P2P]</b-navbar-item>
```

Comment:
shortening this indicator for P2P demos. will take it out entirely once we've implemented exchange_type settings